ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -230,6 +232,34 @@ export default class SecurityTab extends PureComponent {
     );
   }
 
+  renderUse4ByteResolutionToggle() {
+    const { t } = this.context;
+    const { use4ByteResolution, setUse4ByteResolution } = this.props;
+    return (
+      <div ref={this.settingsRefs[9]} className="settings-page__content-row">
```

Comment:
An entry will need to be added here so that this setting is searchable: `ui/helpers/constants/settings.js`

Reply:
Thanks, will do!

Reply:
added in 00c8de220d25ee9263ae2820d083f6bdfff09ac1